armi/reactor/parameters/parameterDefinitions.py

```diff
@@ -55,38 +55,33 @@
 
 
 class Category:
-    """
-    A "namespace" for storing parameter categories.
-
-    Notes
-    -----
-    * `cumulative` parameters are accumulated over many time steps
-    * `pinQuantities` parameters are defined on the pin level within a block
-    * `multiGroupQuantities` parameters have group dependence (often a 1D numpy array)
-    * `fluxQuantities` parameters are related to neutron or gamma flux
-    * `neutronics` parameters are calculated in a neutronics global flux solve
-    * `gamma` parameters are calculated in a fixed-source gamma solve
-    * `detailedAxialExpansion` parameters are marked as such so that they are mapped from the
-       uniform mesh back to the non-uniform mesh
-    * `reactivity coefficients` parameters are related to reactivity coefficient or kinetics
-       parameters for kinetics solutions
-    * `thermal hydraulics` parameters come from a thermal hydraulics physics plugin (e.g., flow
-       rates, temperatures, etc.)
-    """
+    """A "namespace" for storing parameter categories."""
 
     depletion = "depletion"
+    """Parameters used in or calculated by a depletion plugin."""
     cumulative = "cumulative"
+    """Parameters are accumulated over many time steps"""
     cumulativeOverCycle = "cumulative over cycle"
+    """Parameters that are reset at beginning of cycle and accumulated over each cycle."""
     assignInBlueprints = "assign in blueprints"
+    """Parameters that should be assigned in blueprints (e.g., control rod elevation)"""
     retainOnReplacement = "retain on replacement"
     pinQuantities = "pinQuantities"
+    """Parameters are defined on the pin level within a block"""
     fluxQuantities = "fluxQuantities"
+    """Parameters are related to neutron or gamma flux"""
```

Comment:
(This came up in a chat so wanted to put my motivation here)

The change here is to make these docstrings attached to the class constants. They were already documented in the `Category` docstring (some of them at least) but I found two flaws

1. They were documented in the `Notes` section but would make more sense to be properly documented as [class attributes](https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring)
2. Attaching the docstring to the object itself means tools like python, ipython, vscode, etc. can provide context where you are about the thing you're looking at. You don't need to separately pull up the ARMI docs.

I'm not going to die on this hill if people have stronger opinions. This is just my rationale for this change.